impeller/entity/entity_pass.h

```diff
@@ -53,24 +58,57 @@ class EntityPass {
 
   void SetBlendMode(Entity::BlendMode blend_mode);
 
+  void SetBackdropFilter(std::optional<BackdropFilterProc> proc);
+
   std::optional<Rect> GetSubpassCoverage(const EntityPass& subpass) const;
 
   std::optional<Rect> GetElementsCoverage() const;
 
  private:
-  bool RenderInternal(ContentContext& renderer,
-                      RenderTarget render_target,
-                      Point position,
-                      uint32_t pass_depth,
-                      size_t stencil_depth_floor = 0) const;
+  struct EntityResult {
+    /// @brief  The resulting entity that should be rendered. If `std::nullopt`,
+    ///         there is nothing to render.
+    std::optional<Entity> entity = std::nullopt;
+    /// @brief  This is set to `false` if there was an unexpected rendering
+    ///         error while resolving the Entity.
+    bool success = false;
```

Comment:
Can this just be an enum with kSuccess, kFailure, and kSkip?

Reply:
Done.